src/SharedComponents/AstroVirtualAssistant/AstroVirtualAssistant.tsx

```diff
@@ -1,32 +1,60 @@
 import React, { FunctionComponent, useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
 
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 import { Stack, StackItem } from '@patternfly/react-core';
+import { useFlag } from '@unleash/proxy-client-react';
 
 import { Status, useAstro } from '../../Components/AstroChat/useAstro';
 import './astro-virtual-assistant.scss';
 import { AstroChat } from '../../Components/AstroChat/AstroChat';
 import { AstroBadge } from '../../Components/AstroAvatar/AstroBadge';
 import { commandMessageProcessor } from './CommandMessageProcessor';
-import { createPortal } from 'react-dom';
 
 const messageProcessors = [commandMessageProcessor];
 
-export const AstroVirtualAssistant: FunctionComponent = () => {
+interface AstroVirtualAssistantProps {
+  showAssistant: boolean;
+  isOpen: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  startInput?: string;
+}
+
+export const AstroVirtualAssistant: FunctionComponent<AstroVirtualAssistantProps> = ({ showAssistant, isOpen, setOpen, startInput }) => {
   const chrome = useChrome();
   const { messages, setMessages, ask, start, status, error, loadingResponse } = useAstro(messageProcessors, {
     isPreview: chrome.isBeta(),
     auth: chrome.auth,
   });
-  const [isOpen, setOpen] = useState<boolean>(false);
   const [isFullScreen, setFullScreen] = useState<boolean>(false);
 
+  const [input, setInput] = useState<string>('');
+
+  const isOpenConfig = useFlag('platform.virtual-assistant.is-open-config');
+  // if useOpenConfig is enabled, set showAssistant to true, override parameter isOpen and setOpen
+  const [tempIsOpen, setTempIsOpen] = useState(isOpen);
+  if (isOpenConfig) {
+    showAssistant = true;
+    isOpen = tempIsOpen;
+    setOpen = setTempIsOpen;
+  }
+
   useEffect(() => {
     if (isOpen) {
       void start();
     }
   }, [isOpen]);
 
+  useEffect(() => {
+    if (startInput) {
+      setInput(startInput);
+    }
+  }, [startInput]);
+
+  if (!showAssistant) {
+    return null;
```

Comment:
@justinorringer won't this always return null with the feature flag off now?

Reply:
Fixed now, tested locally with flag on and off :)